CHANGELOG.md

```diff
@@ -12,9 +12,10 @@
 
 
 ### Bugfixes
--Removes inactive danger areas at EIDW [#562](https://github.com/openscope/openscope/issues/562)
+- Removes inactive danger areas at EIDW [#562](https://github.com/openscope/openscope/issues/562)
```

Comment:
Be sure this branch includes only the changes for this issue.  This Changelog entry looks like it's from your other branch, please remove it.